packages/gatsby-theme-docs/src/components/anchor.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+
+const A = styled.a``;
+
+const Anchor = (props) => <A name={props.name} />;
```

Comment:
Why do we need a React component when you can just use HTML/JSX in MDX?

```
<a id="foo" />
```

This is valid JSX 👆

Reply:
I am not sure if @nkuehn has mentioned this to you but during my discussions with him, he has said something to the effect of the new docs tool will disallow html. Maybe discuss with him for details.

Also, one main requirement is that we only want to allow the `name` attribute to be used for this purpose.

Reply:
Hmm I see...